content/en/serverless/installation/dotnet.md

```diff
@@ -23,32 +23,34 @@ After you have [installed the AWS integration][1], use .NET to instrument your a
 
 ## Configuration
 
-### Enable AWS X-Ray
+### Configure the Function
 
-Navigate to the Lambda function in the AWS console you want to instrument. In the debugging and error handling section, check the box to enable AWS X-Ray.
+1. Enable [AWS X-Ray active tracing][2] for your Lambda function.
 
-**Note**: If you’re using a Customer Master Key to encrypt traces, add the `kms:Decrypt` method to your IAM policy where the Resource is the Customer Master Key used for X-Ray.
+### Subscribe the Datadog Forwarder to the Log Groups
 
-### Install the AWS X-Ray client library
+You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.
 
-Install the [AWS X-Ray client library][2].
+1. [Install the Datadog Forwarder if you haven't][3].
+2. [Ensure the option DdFetchLambdaTags is enabled][4].
+3. [Subscribe the Datadog Forwarder to your function's log groups][5].
 
-### Subscribe the Forwarder to log groups
+## Quick Start
```

Comment:
@apigirl This is a V1 for what we'll put in "quick start" - let's chat about what we can put here later this week.